kuma/wiki/helpers.py

```diff
@@ -22,7 +21,7 @@
 
 from kuma.core.urlresolvers import reverse
 from .constants import DIFF_WRAP_COLUMN
-from .models import Document
+from .models import Document, memcache
```

Comment:
get_cache in old Django's had the problem of creating cache clients all the time, so the fewer we do that the less connections are open against the memcache cluster